lib/subscriber_list_mover.rb

```diff
@@ -42,8 +42,6 @@ def call
         )
 
         if subscribed_to_destination_subscriber_list.nil?
-          puts "Moving #{subscriber.address} with ID #{subscriber.id} to #{destination_subscriber_list.title} list"
```

Comment:
Could we leave it in but just take out the subscriber address? I e.g. quite like having some output that I can put in a trello ticket for an audit trail in case we need it

Reply:
Ah, we actually get a loooot of output with this task without this line, from [here](https://github.com/alphagov/email-alert-api/blob/df756cf9d320e80752b26b33f01835000cf69067/lib/subscriber_list_mover.rb#L55). Example output:

```
Bad subscriptions count for prefix 'topic':2334
Running migration...
185 active subscribers moving from safeguarding-children to schools-colleges-and-children-s-services-safeguarding-children
185 active subscribers moved from safeguarding-children to schools-colleges-and-children-s-services-safeguarding-children.
32 active subscribers moving from mot-manuals-and-special-notices to mot-manuals
Migration complete
There are 159 remaining bad subscriptions for 'topic' lists.
These pages still have bad lists: ["/topic/immigration-operational-guidance/immigration-rules", "/topic/immigration-operational-guidance/asylum-policy", "/topic/immigration-operational-guidance/fees-forms", "/topic/immigration-operational-guidance/nationality-guidance", "/topic/immigration-operational-guidance/entry-clearance-guidance", "/topic/immigration-operational-guidance/modernised-guidance", "/topic/immigration-operational-guidance/enforcement", "/topic/immigration-operational-guidance/windrush-scheme"].

So unless you feel strongly, I would prefer to remove this additional output.

```